android/build.gradle

```diff
@@ -15,12 +15,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion androidCompileSdkVersion
+    buildToolsVersion androidBuildToolsVersion
 
     defaultConfig {
-        minSdkVersion 14
-        targetSdkVersion 23
+        minSdkVersion androidMinSdkVersion
+        targetSdkVersion androidTargetSdkVersion
```

Comment:
Thanks for the link!

commit a4cfae461086874ff079f52ce35968c11e136bca
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Mar 7 22:38:32 2017 +0000

    Remove specification of targetSdk property for library module
    
    - https://github.com/novoda/notils/pull/76#discussion_r104643557